### PR TITLE
Fr translation update

### PR DESCRIPTION
**Pull request by LapinFou**
- Translated new fields.
- For consistency, all **folder** keywords have been translated to **dossier** (**répertoire** is not used anymore).
- I translated **Style**/**Style sheet** to simply **Style**. Another choice would have been to used the word **template**/**template de style**. But I tried to keep the translation as short as possible. Any comment is welcome.
- Not sure about the translation of **F.In**/**F.Out**.
- I don't know what **Min.Call** is referring to. So, I didn't translate this field.

### CODE REVIEW

File: companion/src/translations/companion_fr.ts [hunk 1]
```diff
@@ -8819,433 +8824,433 @@ Mixage actif par défaut si non-renseigné.</translation>
...
     <message>
-        <location filename="../multimodelprinter.cpp" line="354"/>
+        <location filename="../multimodelprinter.cpp" line="331"/>
         <source>Countdown</source>
         <translation>Compte à rebours</translation>
     </message>
     <message>
-        <location filename="../multimodelprinter.cpp" line="354"/>
-        <source>Minute call</source>
-        <translation>Annonces minutes</translation>
```
Comment (kilrah) [inline companion/src/translations/companion_fr.ts:8948]: minutes c'était ici

File: companion/src/translations/companion_fr.ts [hunk 2]
```diff
@@ -8819,433 +8824,433 @@ Mixage actif par défaut si non-renseigné.</translation>
...
-        <location filename="../multimodelprinter.cpp" line="442"/>
+        <location filename="../multimodelprinter.cpp" line="331"/>
+        <location filename="../multimodelprinter.cpp" line="419"/>
         <source>Switch</source>
         <translation>Inter</translation>
     </message>
     <message>
-        <location filename="../multimodelprinter.cpp" line="442"/>
-        <source>Fade IN</source>
-        <translation>Fondu en entrée</translation>
```
Comment (kilrah) [inline companion/src/translations/companion_fr.ts:8866]: Fondu ici

**Comment by kilrah** [comment]
Style sheet = Feuille de style en FR

**Comment by LapinFou** [comment]
@kilrah 
Hi! 😄 
Sorry, but I don't understand what you're highlighting.
Could you, please, clarify the point?

File: companion/src/translations/companion_fr.ts (hunk 2 above)
Comment (raphaelcoeffic) [reply companion/src/translations/companion_fr.ts:8866]: Et Fromage en dessert?

Comment (LapinFou) [reply companion/src/translations/companion_fr.ts:8866]: Ha ben, c'est malin ça.... 😋

Comment (kilrah) [reply companion/src/translations/companion_fr.ts:8866]: Le F.In c'est ca, qui était "Fade IN" avant. Donc la traduction est la même. Même chose pour le minute call, il a simplement été raccourci en anglais. A toi de voir si tu veux aussi raccourcir en français, mais comme ça tu sais ce que c'est...

Comment (LapinFou) [reply companion/src/translations/companion_fr.ts:8866]: Haaaa OK !
Compris.
Je vais faire la modif demain.
Merci.

**Comment by LapinFou** [comment]
PR should now be ready to merge.

**Pull request merged**